### DISCUSSION TITLE
Adding external resistance option

### DISCUSSION THREAD

**Discussion by Naroek56** [pybamm-team/PyBaMM#1849] [Q&A] [answered]
For cell modelling it would be helpful to add a term that takes into account external series resistances such as tab weld resistances.
Would it be possible to add an optional resistance term to the simulation?

Thanks in advance!

**Comment by brosaplanella** [comment, marked as answer]
If you just want to add the resistance to the voltage you can just post-process the voltage values by doing
```python3
V = solution["Terminal voltage [V]"] - R * solution["Current [A]"]
```

For more complex setups, you might want to use PyBaMM's sibling package `liionpack` (see https://github.com/pybamm-team/liionpack)

This is a relevant question, so I am converting it into a discussion.

**Reply by valentinsulzer** [reply]
For CV control it would be useful to have this as a pre-processing feature

**Reply by brosaplanella** [reply]
Do we have an issue for this already?

**Reply by rtimms** [reply]
Can we just add it [here](https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/models/submodels/electrode/base_electrode.py#L147)?
